tests/integration/Mvc/Model/BinderCest.php

```diff
@@ -0,0 +1,500 @@
+<?php
+
+namespace Phalcon\Test\Integration\Mvc\Model;
+
+use IntegrationTester;
+use Phalcon\Cache\Backend\Apc;
+use Phalcon\Cache\Frontend\Data;
+use Phalcon\Di;
+use Phalcon\Mvc\Dispatcher;
+use Phalcon\Mvc\Micro;
+use Phalcon\Mvc\Micro\Collection;
+use Phalcon\Mvc\Model\Binder;
+use Phalcon\Mvc\Model\Manager;
+use Phalcon\Mvc\Model\MetaData\Memory;
+use Phalcon\Test\Models\People;
+use Phalcon\Test\Models\Robots;
+
+/**
+ * \Phalcon\Test\Integration\Mvc\Model\BindingCest
+ * Tests the Phalcon\Mvc\Application component
+ *
+ * @copyright (c) 2011-2016 Phalcon Team
+ * @link          http://www.phalconphp.com
+ * @author        Andres Gutierrez <andres@phalconphp.com>
+ * @author        Serghei Iakovlev <serghei@phalconphp.com>
+ * @author        Wojciech Ślawski <jurigag@gmail.com>
+ * @package       Phalcon\Test\Integration\Mvc\Model
+ *
+ * The contents of this file are subject to the New BSD License that is
+ * bundled with this package in the file docs/LICENSE.txt
+ *
+ * If you did not receive a copy of the license and are unable to obtain it
+ * through the world-wide-web, please send an email to license@phalconphp.com
+ * so that we can send you a copy immediately.
+ */
+class BinderCest
+{
+    /**
+     * @var Apc
+     */
+    private $cache;
+
+    /**
+     * @var Binder
+     */
+    private $modelBinder;
+
+    /**
+     * @var Manager
+     */
+    private $modelsManager;
+
+    /**
+     * Executed before each test
+     *
+     * @param IntegrationTester $I
+     */
+    public function _before(IntegrationTester $I)
+    {
+        Di::setDefault($I->getApplication()->getDI());
+
+        $this->cache = new Apc(new Data(['lifetime' => 20]));
+        $this->modelBinder = new Binder($this->cache);
+
+        $this->modelsManager = $I->getApplication()->getDI()->getShared('modelsManager');
+
+        $I->haveServiceInDi('modelsMetadata', function () {
+            return new Memory;
+        }, true);
+    }
+
+    /**
+     * Tests dispatcher and model binder
+     *
+     * @author Wojciech Ślawski <jurigag@gmail.com>
+     * @since  2016-10-20
+     *
+     * @param IntegrationTester $I
+     */
+    public function testDispatcher(IntegrationTester $I)
```

Comment:
could you name it in a bit more meaningful way? I'd really like to know what actual functionality is tested